src/Microsoft.Extensions.Configuration/ConfigurationSection.cs

```diff
@@ -57,6 +74,11 @@ public string Value
             }
         }
 
+        /// <summary>
+        /// Gets or sets a configuration value.
```

Comment:
`Gets or sets the value corresponding to a configuration key` (I get that this may be inherited all the way from `IConfiguration`.